static/css/zamboni/developers.css

```diff
@@ -702,6 +702,8 @@ div.history-container {
 div.review-entry {
     padding: 0.5em 1em;
     margin-bottom: 0.25em;
+    display: flex;
```

Comment:
is default alignment working? Feel like it rarely does..

Reply:
I mean, the css is working as expected for me?

With:
![image](https://github.com/user-attachments/assets/ce5ba726-cc78-4766-8610-458d00fc5798)

Without:
![image](https://github.com/user-attachments/assets/c9dfaded-b47a-485c-b873-79c3489903f8)